main.css

```
@@ -0,0 +1,928 @@
+    }
+    .one , .three{
+        margin: 0;
+    }
+    .one {
+        background-image: url("./assets/images/Shape\ 2\ \(mob\).svg");
+        background-position: 45px 29px;
+        background-repeat: no-repeat;
+    }
+    .two {
+        background-image: url("./assets/images/Shape\ 3\ \(mob\).svg");
+        background-position: 76px 25px;
+        background-repeat: no-repeat;
+    }
+    .three {
+        background-image: url("./assets/images/Shape\ 4\ \(mob\).svg");
+        background-position: 68px 25px;
+        background-repeat: no-repeat;
+    }
+    .section1 {
+        background: #ffffff;
+    }
+    .section1 {
+        height: 484px;
+    }
+    .section1-wrapper {
+        padding: 10px;
+        margin-top: 4px;
+        display: grid;
+        grid-gap: 10px;
+        width: 100%;
+        height: 100%;
+        grid-template-columns: 298px 298px;
+        grid-template-rows: 232px 232px;
+    }
+    .section1-box1 {
+        /* margin-left: 12px;         */
+        width: 298px;
+        margin: 2px 10px 0 0;
+    }
+    
+    .section2-box2 {
+    }
+    .section1-box3 { 
+        margin-left: -12px;
+        width: 642px;
+        height: 232px;
+        background: #f5c507;
+        background-image: url("./assets/images/twitter.svg");
+        background-position: 85px 57px;
+        background-repeat: no-repeat;
+        
+    }
+    .section1-box1-content {
+        word-break: break-all;        
+        margin: 16px 27px;
+        width: 243px;
+        height: 198px;
+        font-size: 29px;
+        letter-spacing: 0px;
+        line-height: 33px;
+        color: #ffffff;
+        font-family: "Segoe WP";
+        font-weight: 300;
+    }
+    .plus {
+        position: absolute;
+        margin-top: 201px;
+        margin-left: 273px;
+    }
+
+    .section2 {
+        background: #474747;
+        height: 466px;
+    }
+    .section2-box1 {
+        margin: 0;
+    }
+    .messages {
+        display: block;
+        margin-left: 319px;
+        margin-top: 23px;
+        font-size: 13px;
+        letter-spacing: 0px;
+        color: #8f8e8e;
+        font-family: "Arial";
+    }
+    .message1 {
+        margin-top: 5px;
+        background-image: url("./assets/images/Message Rectangle(1).png");
+        background-repeat: no-repeat;
+        height: 40px;
+        padding-top: 7px;
+        padding-right: 54px;
+        padding-left: 37px;
+        margin-top: 7px;
+        margin-bottom: 7px;
+    }
+    .message {
+        margin-top: 7px;
+        margin-bottom: 7px;
+        background-image: url("./assets/images/Message Rectangle.png");
+        background-repeat: no-repeat;
+        height: 40px;
+        height: 40px;
+        padding-top: 7px;
+        padding-right: 54px;
+        padding-left: 37px;
+        /* margin-top: 23px; */
+    }
+    
+    .myInput {
+        margin-left: 15px;
+        width: 200px;
+        height: 29px;
+        border-radius: 12px 0px 0px 12px;
```

use zero values without units
```suggestion
        border-radius: 12px 0 0 12px;
```